ShipingCost.js

```diff
@@ -17,4 +17,4 @@ function calshipingCost(cost, location) {
     }
 
 }
-calshipingCost(30, 'US');
\ No newline at end of file
+calshipingCost(40, 'US');
```

Comment:
returning console.log will return the output of the console.log function, which is undefined. You should probably just return the string and console log it where you call the function